apps/docs/content/components/collections/Listbox.mdx

```diff
@@ -1,48 +1,183 @@
 ---
-title: ListBox (WIP)
-description: TBD
+title: ListBox
+description: A ListBox is a disclosure component that appears with a set of actions relevant to a specific control, interface area, data element or application view. Typically, this context is determined by the user’s current selection prior to invoking the menu. Listbox can be opened from components such as Selects or Buttons.
 category: "collections"
 links:
     source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/listbox/src/ListBox.tsx
 ---
 
+<Example src="ListBox/docs/preview" isOpen />
+
 ## Props
 
-TODO: Add the props table, if we have multiple props table (ex: TagList and Tag) in the same file,  use the name of the component in ### before the each props table
+<PropTable component="ListBox" />
+
+### ListBoxItem
+
+<PropTable component="ListBoxItem" />
+
 
-## Guidelines
+<FeatureFlag flag="next">
+    ## Guidelines
 
-TODO: If we have some guidelines about this component's usage
+    TODO: If we have some guidelines about this component&apos;s usage
 
-### Accessibility ?
+    ### Accessibility ?
 
-TODO: If we have some guidelines about this component and accessibility
+    TODO: If we have some guidelines about this component and accessibility
+</FeatureFlag>
 
 ## Anatomy
 
-TODO: We have anatomy screenshots from the Figma, we could most likely use them here
+<FeatureFlag flag="rc">
+    TODO: We have anatomy screenshots from the Figma, we could most likely use them here
 
-### Concepts
+    ### Concepts
 
-TODO: links to related concepts
+    TODO: links to related concepts
+</FeatureFlag>
 
 ### Composed Components
 
-TODO: links to related components (most likely all component which are used by the slots)
+- [Divider](./Divider)
+- [ListBoxItem](./ListBoxItem)
```

Comment:
This goes to a 404 page

Reply:
could probably just remove the link. Maybe we should have composed components for ListBoxItem as well.
Also, you are Missing Section for LIstBox.
Inside section, you can have a Header as well.

Reply:
you could also try linking to within the page for ListBoxItem.